wasm/src/language_service.rs

```diff
@@ -196,6 +196,30 @@ impl LanguageService {
             .collect()
     }
 
+    pub fn get_format_changes(&self, uri: &str) -> Vec<ITextEdit> {
+        let edits = self.0.get_format_changes(uri);
+        edits
+            .into_iter()
+            .map(|edit| {
+                // ToDo: is there an easier way to convert data types?
```

Comment:
Yes, you should be able to say `range: edit.range.into()` , see the other usage of `TextEdit` in this file

Reply:
done